core/build.gradle.kts

```diff
@@ -61,5 +59,6 @@ dependencies {
     androidTestImplementation(libs.androidx.test.espresso.core)
     androidTestImplementation(libs.androidx.test.junit)
     androidTestImplementation(libs.compose.ui.test.junit4)
+    androidTestImplementation(libs.compose.foundation)
```

Comment:
We only use foundation APIs (in this case Modifier.background) in our instrumentation tests